src/Associations.cc

```diff
@@ -85,6 +89,33 @@ void Associations::setCommonTangentPoint(lsst::afw::geom::Point2D const &commonT
     for (auto &ccdImage : ccdImageList) ccdImage->setCommonTangentPoint(_commonTangentPoint);
 }
 
+lsst::sphgeom::Circle Associations::computeBoundingCircle() const {
+    // Compute the frame on the common tangent plane that contains all input images.
+    Frame tangentPlaneFrame;
+
+    for (auto const &ccdImage : ccdImageList) {
+        Frame CTPFrame = ccdImage->getPixelToCommonTangentPlane()->apply(ccdImage->getImageFrame(), false);
+        if (tangentPlaneFrame.getArea() == 0)
+            tangentPlaneFrame = CTPFrame;
+        else
+            tangentPlaneFrame += CTPFrame;
+    }
+
+    // Convert tangent plane coordinates to RaDec.
+    AstrometryTransformLinear identity;
+    TanPixelToRaDec commonTangentPlaneToRaDec(identity, _commonTangentPoint);
+    Frame raDecFrame = commonTangentPlaneToRaDec.apply(tangentPlaneFrame, false);
+
+    std::vector<sphgeom::UnitVector3d> points;
+    points.reserve(4);
+    points.emplace_back(sphgeom::LonLat::fromDegrees(raDecFrame.xMin, raDecFrame.yMin));
+    points.emplace_back(sphgeom::LonLat::fromDegrees(raDecFrame.xMax, raDecFrame.yMin));
+    points.emplace_back(sphgeom::LonLat::fromDegrees(raDecFrame.xMin, raDecFrame.yMax));
+    points.emplace_back(sphgeom::LonLat::fromDegrees(raDecFrame.xMax, raDecFrame.yMax));
+
```

Comment:
What confused me here is that I guess `x` and `y` in the `raDecFrame` are in sky coordinates?  It is not apparent from the `x` and `y` terminology (outside the scope of this ticket!) but maybe it's worth adding a comment to this effect?  (Or it might be well-known to those more familiar with how the wcs `Frame`s work.)

Reply:
Yes, this is confusing because jointcal's internal objects (`Frame` and `Point`) are used in both on-sky, and on-sensor cases, with `x` and `y` having different meanings in those cases. This is unfortunate, but I haven't had time to do the plumbing work to fix it (again, DM-4044).

I've added a comment that may clarify things?